skops/io/_utils.py

```diff
@@ -171,3 +171,21 @@ def get_state(value, save_context):
 
     res["__id__"] = __id__
     return res
+
+
+def get_type_name(
+    t: Any,
+) -> str:
+    """Simple helper function to take in a type, and return its name as a string"""
```

Comment:
```suggestion
    """Helper function to take in a type, and return its name as a string"""
```

I think "simple" doesn't add anything here. I'm guilty of using words like "simple" myself, but I learned recently that they should be avoided in documentation :)